src/components/automated/Conversation.vue

```diff
@@ -1,32 +1,47 @@
 <template>
-    <div class="conversation-container">
-        <chain-node v-model="chain"></chain-node>
-        <div>
-            <span @click="handelSave">Save</span>
-        </div>
+  <div class="conversation-container">
+    <chain-node v-model="chain"></chain-node>
+    <div>
+      <span @click="handelSave">Save</span>
     </div>
+  </div>
 </template>
 
 <script>
-    import tree from './models'
-    import ChainNode from './ChainNode'
-    export default {
-        name: 'conversation',
-        components: {
-            'chain-node': ChainNode,
-        },
-        data() {
-            return {
-                chain: tree,
-            }
-        },
-        methods: {
-            handelSave() {
-                console.log("chain:",JSON.stringify(this.chain));
-            }
-        }
-    }
+import tree from "./models";
```

Comment:
when you reformat the file, many changes will appear.
- don't use auto format
- format the code manually if possible
- check your files before commit (using Visual code)

![image](https://user-images.githubusercontent.com/16289510/84517372-e3527700-acd7-11ea-9e27-bee1cce392a1.png)
- the changes must be your code only.